Docs/beta/snippets/torch/apply_adaround.py

```diff
@@ -2,7 +2,7 @@
 # =============================================================================
 #  @@-COPYRIGHT-START-@@
 #
-#  Copyright (c) 2024, Qualcomm Innovation Center, Inc. All rights reserved.
+#  Copyright (c) 2023, Qualcomm Innovation Center, Inc. All rights reserved.
```

Comment:
minor: We should revert this change and keep it as 2024.
We can handle it in a separate PR